app/src/main/java/com/nextcloud/talk/controllers/LocationPickerController.kt

```diff
@@ -213,7 +213,7 @@ class LocationPickerController(args: Bundle) :
         return true
     }
 
-    @Suppress("Detekt.TooGenericExceptionCaught")
+    @Suppress("Detekt.TooGenericExceptionCaught", "Detekt.ComplexMethod")
```

Comment:
still needed since you refactored the method?

Reply:
Yes. The method is still too complex Detekt says. But I can live with it.

But if you insist I will look again on it :-)

Reply:
Nah, all good then 👍